test/engine.spec.ts

```diff
@@ -556,7 +556,7 @@ describe('#getCellType', () => {
   it('formula matrix', () => {
     const engine = HyperFormula.buildFromArray([['=TRANSPOSE(C1:C2)']])
 
-    expect(engine.getCellType(adr('A1'))).toBe(CellType.ARRAY)
+    expect(engine.getCellType(adr('A1'))).toBe(CellType.ARRAYFORMULA)
     expect(engine.getCellType(adr('B1'))).toBe(CellType.ARRAY)
   })
```

Comment:
Can we add test with an array that is not a result of a formula? With `{1,2,3;4,5,6}` we will have `CellType.ARRAY` returned for all cells, A1 included, right?

Reply:
I don't think it is possible to create array without formula.

Reply:
Ok, I forgot that `=` is needed

https://github.com/handsontable/hyperformula/blob/413e199a632452bffb45f33c747b53a6f8af2cab/test/arrays.spec.ts#L234-L237